utilities/src/main/java/io/syndesis/qe/templates/PublicOauthProxyTemplate.java

```diff
@@ -45,24 +45,30 @@ public static void deploy() {
     public static void cleanUp() {
         log.info("Cleaning up everything for {} template", TEMPLATE_NAME);
 
+        OpenShiftUtils.getInstance().getDeploymentConfigs().stream().filter(dc -> "syndesis-public-oauthproxy".equals(dc.getMetadata().getName()))
+            .findFirst()
+            .ifPresent(dc -> OpenShiftUtils.getInstance().deleteDeploymentConfig(dc, true));
+        OpenShiftUtils.getInstance().getServices().stream().filter(service -> "syndesis-public-oauthproxy".equals(service.getMetadata().getName()))
```

Comment:
when you have the name of the resource, you can directly use `OpenShiftUtils.getInstance().<resource method>.withName("").delete()`